benches/crdt.md

```diff
@@ -0,0 +1,1097 @@
+---
```

Comment:
Where did you get this file from? I think if you got it from https://github.com/xi-editor/xi-editor/, then we probably need to make some acknowledgement of that fact due to the Apache 2.0 license that repository seems to be released under.

Maybe file it under third_party/xi-editor similar to what is already there at third_party/CommonMark?